src/components/molecules/Post/Post.jsx

```diff
@@ -0,0 +1,65 @@
+import { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+// import { Form } from 'formik';
+import { Text, ImageComponent } from '../..';
+import { routes } from '../../../routes/Routes';
+import { Icon } from '../../atoms/IconComponent/Icon';
+import { dateFormat } from '../../../utils/format';
+
+import './Post.scss';
+
+export const Post = ({ children, date, name, avatar, profileID }) => {
+    const navigate = useNavigate();
+    const redirect = useCallback(() => {
+        navigate(routes.Profile);
+    }, [navigate]);
+
+    return (
+        <div className='post__field'>
+            <ImageComponent
+                src={avatar}
+                size='small'
+                onClick={useCallback(() => {
+                    redirect(`${routes.Feed}${routes.Profile}/${profileID}`);
+                }, [profileID, redirect])}
+            />
+            <Text customClass='username' type='accent' size='medium'>
+                {name}
+            </Text>
+            <Text size='small' customClass='date' type='primary'>
+                {dateFormat(date)}
+            </Text>
+            <Text type='primary' customClass='message' size='medium'>
+                {children}
+            </Text>
+            <Icon
+                iconName='like'
+                // onClick={}
+                size='medium'
+                className='like_button'
+            />
+            <Icon
```

Comment:
these icons should be clickable and should change the state (filled and not) ~ this should be done via the API, but you can prepare logic for that

Reply:
to create it we don't have to add "filled" version of icons as another component? Then, when clicked, the icon will change and the possible counter will increase